unitTests/ui/ResultListTest.ts

```diff
@@ -819,7 +856,7 @@ export function ResultListTest() {
             test.cmp.displayMoreResults(50).then(() => {
               Simulate.query(test.env);
               setTimeout(() => {
-                expect(infiniteScrollContainer.scrollTop).toBe(500);
+                expect(scrollTopSpy).not.toHaveBeenCalled();
```

Comment:
Here also, the test name does not really match the test content.

I see they are similar in nature, but not "exactly" testing the same thing...